src/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -927,7 +927,7 @@ public enum DataViewRowState
         OriginalRows = 42,
         Unchanged = 2,
     }
-    [System.ComponentModel.TypeConverter(typeof(System.ComponentModel.ExpandableObjectConverter))]
+    [System.ComponentModel.TypeConverterAttribute(typeof(System.ComponentModel.ExpandableObjectConverter))]
```

Comment:
Please review: @divega, @ajcvickers, @afsanehr, @david-engel, @tarikulsabbir

Reply:
@ahsonkhan is there something specific that you wanted to be looked at or are you asking for the whole whole namespace to be reviewed? 

FWIW, this can be time consuming because of all the noise caused by using different ways to express the same thing in code, but I see hardly any meaningful changes.